components/gitpod-cli/cmd/env.go

```diff
@@ -213,6 +227,11 @@ func setEnvs(ctx context.Context, scopeUser bool, args []string) error {
 		g.Go(func() error {
 			err = result.client.SetEnvVar(ctx, v)
 			if err != nil {
+				if ferr, ok := err.(*jsonrpc2.Error); ok && ferr.Code == http.StatusForbidden && setEnvScope == envScopeUser {
+					return fmt.Errorf(""+
+						"Can't automatically create Env Var `%s` for security reasons.\n"+
```

Comment:
Out of curiosity, why the capitalization?

Reply:
Fixed. I copied the text from a suggestion.